crates/pixi_build_types/src/project_model.rs

```diff
@@ -43,6 +43,15 @@ impl VersionedProjectModel {
         // increase this when adding a new version
         1
     }
+
+    /// Convert the versioned project model to a specific version.
+    pub fn v1(self) -> Option<ProjectModelV1> {
```

Comment:
Lets use `into_v1` and maybe we can also add `as_v1` which returns a reference?

Reply:
Good one!